USAGE.md

```diff
@@ -5013,7 +5013,7 @@ puts response.headers
 **This endpoint allows you to associate a link whitelabel with a subuser account.**
 
 Link whitelables can be associated with subusers from the parent account. This functionality allows
-subusers to send mail using their parent's linke whitelabels. To associate a link whitelabel, the parent account
+subusers to send mail using their parent's like whitelabels. To associate a link whitelabel, the parent account
```

Comment:
Can you please change this to "Link whitelabels" ?

Reply:
Done!
Thanks, @mbernier!